test/ui/CategoryFacet/CategoryFacetTest.ts

```diff
@@ -4,14 +4,45 @@ import { $$ } from '../../Test';
 
 export function CategoryFacetTest() {
   describe('CategoryFacet', () => {
-    let categoryFacet;
+    let categoryFacet: CategoryFacet;
 
     beforeEach(() => {
-      categoryFacet = Mock.basicComponentSetup<CategoryFacet>(CategoryFacet);
+      categoryFacet = Mock.basicComponentSetup<CategoryFacet>(CategoryFacet).cmp;
     });
     it('calling hide adds the coveo hidden class', () => {
-      categoryFacet.cmp.hide();
-      expect($$(categoryFacet.cmp.element).hasClass('coveo-hidden')).toBeTruthy();
+      categoryFacet.hide();
+      expect($$(categoryFacet.element).hasClass('coveo-hidden')).toBeTruthy();
     });
+
+    describe('calling changeActivePath', () => {
+      let newPath: string[];
+      let queryPromise: Promise<void>;
+      beforeEach(() => {
+        newPath = ['new', 'path'];
+        queryPromise = categoryFacet.changeActivePath(newPath);
+      });
+
+      it('sets the new path', () => {
+        expect(categoryFacet.activePath).toEqual(['new', 'path']);
+      });
+
+      it('triggers a new query', () => {
+        expect(categoryFacet.queryController.executeQuery).toHaveBeenCalled();
```

Comment:
Would it be useful to use `toHaveBeenCalledWith` here, or does `executeQuery` not accept any arguments?